webrbiblio.bib

```diff
@@ -29,15 +32,17 @@ @Article{Zghaibeh2008
 }
 
 @inproceedings{locher2006free,
-	title={Free riding in BitTorrent is cheap},
-	author={Locher, Thomas and Moor, Patrick and Schmid, Stefan and Wattenhofer, Roger},
-	year={2006},
-	organization={Citeseer}
+    author = {Thomas Locher and Patrick Moor and Stefan Schmid and Roger Wattenhofer},
+    title = {Free Riding in BitTorrent is Cheap},
+    booktitle = {In Proc. 5th Workshop on Hot Topics in Networks (HotNets)},
+    year = {2006}
 }
 
 @misc{multihash,
 	title={Multihash},
 	howpublished={\url{https://github.com/multiformats/multihash}},
+    author={Benet, Juan},
+    year={2014},
```

Comment:
Where did you find this information? There's at least another github repo that needs this

Reply:
I got it from the first commit in the repo, but I guess that's not necessarily correct in this specific case.
You can remove it (I was trying to add author/year to all entries at some point just to make it compile, but it's not necessary anymore).